services/data_plane_manager/src/main/java/com/futurewei/alcor/dataplane/entity/MulticastGoalStateV2.java

```diff
@@ -20,10 +20,12 @@
 import com.futurewei.alcor.web.entity.dataplane.MulticastGoalStateByte;
 
 import java.util.ArrayList;
+import java.util.HashSet;
 import java.util.List;
+import java.util.Set;
 
 public class MulticastGoalStateV2 {
-    private List<String> hostIps;
+    private Set<String> hostIps;
     private List<String> vpcIds;
     private List<String> topics;
     private List<String> subTopics;
```

Comment:
It is a good change. Could we also apply it to other lists? If the change is too huge, we could create an issue and then do it later.

Reply:
Will do it in next PR.

Reply:
Cool. Issue https://github.com/futurewei-cloud/alcor/issues/716 created so that we don't forget :-)